src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/Solutions/ViewASolution.feature

```diff
@@ -12,30 +12,19 @@ Scenario: Solution Details
 	And Solution Full Description
 	And About Solution URL
 	And Contact Details
-	And Last Updated Date
 	And list of Capabilities
 	And Solution ID
 
-Scenario: Attachment Download Link Displayed
-	Given that a User views a Solution
-	When the User is viewing the Solution Page
-	Then there is a link for the User to download an attachment
-
 Scenario: Foundation Solution Indicator
 	Given that a User views a Foundation Solution
 	When the User is viewing the Solution Page
 	Then the page will contain an indication that the Solution meets the criteria for a Foundation Solution Set
 
 Scenario: Capabilities displayed correctly
-	Given that a User views a Solution
+	Given that a User views a Solution	
 	When the User is viewing the Solution Page
 	Then the capabilities listed match the expected capabilities in the database
 
-Scenario: Download more information
```

Comment:
where is this covered if you're removing it?

Reply:
This was only ever a placeholder until more functionality was included. As it is now obsolete, I have removed it